rfcs/proposed/parallel_block_for_task_arena/README.md

```diff
@@ -129,7 +130,6 @@ class task_arena {
     enum class leave_policy : /* unspecified type */ {
         automatic = /* unspecifed */,
         fast = /* unspecifed */,
-        delayed = /* unspecifed */
     };
```

Comment:
Wouldn't the user still need an explicit `delayed` to override an implementation choosing `fast` on a hybrid system?

Reply:
Or do we just expect the user to always create a parallel phase in such situation?

Reply:
Yes, I think the expectation is to use a parallel phase. I suppose the danger with a "delayed" would be that is less targeted that "fast" or the parallel phase; since it overrides the default for the entire lifetime of the arena. Overriding with "fast" makes sense, since it alters the behavior for each execute call to the arena. Adding some kind of block around a single call would be strange to add "fast".  But "delayed" is about identifying a region that contains multiple execute calls and where there should be stickiness to the arena.